RockWeb/Themes/NewSpring/assets/Lava/WORKFLOWS/kidspring-first-time-follow-up.lava

```diff
@@ -0,0 +1,88 @@
+<h2>First Time Family Follow-Up</h2>
+<p>The <b>{{ Workflow | Attribute:'FamilyGroup' }}</b> registered with KidSpring at the <b>{{ Workflow | Attribute:'Campus' }}</b> campus on <b>{{ Workflow.CreatedDateTime | Date:'M/d/yyyy' }}.</b> Use their contact information below to follow-up with them about their visit, and invite them back next week!</p>
+
+<hr class="push-ends">
+
+<div class="row">
+{% workflow id:'{{ Workflow.Id }}' iterator:'workflows' %}
+    {% for workflow in workflows %}
+    
+        {% assign address = workflow | Attribute:'Address','Object' %}
+        
+        <div class="col-xs-12 col-md-12">
+            <h3 class="push-half-bottom">The {{ workflow | Attribute:'FamilyGroup' }}</h3>
+            {% if address and address != empty %}<p><b>Address:</b><br>{{ address.Street1 }}{% if address.Street2 != empty %}<br>{{ address.Street2 }}{% endif %}<br>{{ address.City }}, {{ address.State }} {{ address.PostalCode }}</p>{% endif %}
+        </div>
+
+        {% workflowactivity where:'ActivityTypeId == 1928 || ActivityTypeId == 1930 && WorkflowId == {{ workflow.Id }}' iterator:'activities' %}
+            {% for activity in activities %}<div class="col-xs-12 col-md-6 push-half-bottom">
+                <div class="well">
+                {% assign adult = activity | Attribute:'Adult','Object' %}
+                {% assign child = activity | Attribute:'Child','Object' %}
+                {% if adult != empty %}
+                    {% assign person = adult %}
+                {% elseif child != empty %}
+                    {% assign person = child %}
+                {% endif %}
+                {% assign grade = Activity | Attribute:'Grade' %}
+                {% assign phone = person | PhoneNumber:'Mobile' %}
+                
+                {% comment %}If child, get most recent attendance from the past week{% endcomment %}
+
+                {% assign attendedRoom = '' %}
+                {% assign attendedService = '' %}
+                
+                {% if activity.ActivityTypeId == 1930 %}
```

Comment:
This line will need to be updated with a new `ActivityTypeId` value once we move the workflow to production.